recipes/pulseaudio/all/conanfile.py

```diff
@@ -6,13 +6,13 @@
 from conan.tools.layout import basic_layout
 import os
 
-required_conan_version = ">=1.50.0"
+required_conan_version = ">=1.53.0"
 
 
 class PulseAudioConan(ConanFile):
     name = "pulseaudio"
     description = "PulseAudio is a sound system for POSIX OSes, meaning that it is a proxy for sound applications."
-    topics = "sound",
+    topics = ("sound",)
```

Comment:
I'm actually not quite sure what `package_type` this would be

Reply:
definitely a library